lib/cisco_node_utils/vlan.rb

```diff
@@ -53,19 +54,36 @@ def destroy
       config_set('vlan', 'destroy', @vlan_id)
     end
 
-    def cli_error_check(result)
+    def cli_error_check(result, ignore_message=nil)
       # The NXOS vlan cli does not raise an exception in some conditions and
       # instead just displays a STDOUT error message; thus NXAPI does not detect
       # the failure and we must catch it by inspecting the "body" hash entry
       # returned by NXAPI. This vlan cli behavior is unlikely to change.
+      # Check for messages that can be safely ignored.
+
+      unless ignore_message.nil?
+        # Check result against ignore_message
+        # Return if  ignore_message is included in result
+        fail result[2]['body'] if
```

Comment:
This is a behavior in 9K where warning and error appear in the same output when trying to associate another isolated vlan to the primary.